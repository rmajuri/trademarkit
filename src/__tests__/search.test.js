import React from 'react';
import renderer from 'react-test-renderer';
import Search from '../search/search';

it('renders App correctly', () => {
  const tree = renderer
    .create(
        <Search searchPhrase="cats" setSearchPhrase={() => setSearchPhrase()}/>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});