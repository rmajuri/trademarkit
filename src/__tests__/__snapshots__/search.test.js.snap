// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App correctly 1`] = `
<div>
  <form>
    <div
      className="MuiFormControl-root MuiTextField-root"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled Mui-required Mui-required"
        data-shrink={true}
      >
        Search trademarks
        <span
          className="MuiFormLabel-asterisk MuiInputLabel-asterisk"
        >
          â€‰
          *
        </span>
      </label>
      <div
        className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiOutlinedInput-input"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          placeholder="Enter a brand name idea to see relevant trademark information"
          required={true}
          type="text"
          value="cats"
        />
        <fieldset
          aria-hidden={true}
          className="PrivateNotchedOutline-root-62 MuiOutlinedInput-notchedOutline"
          style={
            Object {
              "paddingLeft": 8,
            }
          }
        >
          <legend
            className="PrivateNotchedOutline-legend-63"
            style={
              Object {
                "width": 0,
              }
            }
          >
            <span
              dangerouslySetInnerHTML={
                Object {
                  "__html": "&#8203;",
                }
              }
            />
          </legend>
        </fieldset>
      </div>
    </div>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text  makeStyles-root-1"
      disabled={false}
      label="submit"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="submit"
    >
      <span
        className="MuiButton-label"
      >
        submit
      </span>
    </button>
  </form>
</div>
`;
